cpp/clang/.clang-format

```diff
@@ -0,0 +1,141 @@
+---
+AccessModifierOffset: -4
+AlignAfterOpenBracket: Align
+AlignArrayOfStructures: Right
+AlignConsecutiveAssignments: false
+AlignConsecutiveBitFields: None
+AlignConsecutiveDeclarations: false
+AlignConsecutiveMacros: AcrossEmptyLinesAndComments
+AlignEscapedNewlines: Left
+AlignOperands: true
+AlignTrailingComments: Always
+AllowAllArgumentsOnNextLine: true
+AllowAllParametersOfDeclarationOnNextLine: true
+AllowShortBlocksOnASingleLine: Empty
+AllowShortCaseLabelsOnASingleLine: false
```

Comment:
Can it be `false`?
https://clang.llvm.org/docs/ClangFormatStyleOptions.html#allowshortifstatementsonasingleline

Reply:
yes it can thanks to backward compatibility but you're right, better to change it to new type

Reply:
if I got you right way you pinned a wrong link, here is the right one with boolean type: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#:~:text=0%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D%3B-,AllowShortCaseLabelsOnASingleLine,-(Boolean)

Reply:
Oops. I left comment in wrong place. But you can find AllowShortIfStatementsOnASingleLine in your config and it's false

Reply:
Changed to `Never`